app/src/main/java/com/door43/translationstudio/App.java

```diff
@@ -797,6 +797,35 @@ public static void clearTargetTranslationSettings(String targetTranslationId) {
     }
 
     /**
+     * moves all settings for a target translation to new target translation (such as when target language changed)
+     * @param targetTranslationId
+     */
+    public static void moveTargetTranslationSettings(String targetTranslationId, String newTargetTranslationId) {
```

Comment:
The `Migration` class is just for performing app migrations not content migrations. The `TargetTranslationMigrator` is used to migrate the content.

You are correct we do not want to reference App from the `Migration` class. Instead we can pass in a reference to `SharedPreferences`.